config-default.sh

```diff
@@ -29,3 +29,14 @@ CLIENT_PUBLIC_KEY_PATH=${CLIENT_PUBLIC_KEY_PATH:-~/.ssh/id_rsa.pub}
 
 # Max time period for stack provisioning. Time in minutes.
 STACK_CREATE_TIMEOUT=${STACK_CREATE_TIMEOUT:-60}
+
+# Private Docker registry url. Leave empty for default docker hub.
+DOCKER_REGISTRY=${DOCKER_REGISTRY:-}
+
+ETCD_IMAGE=${ETCD_IMAGE:-gcr.io/google_containers/etcd-amd64:2.2.1}
```

Comment:
I would rather suggest to use prefix. We test a fixed combination of images and versions. End user is not supposed to change it.